living/sa.md

```diff
@@ -4,7 +4,30 @@ Additional names:
 - 
 
 News:
-- 
+- https://transcontinental.com.au/
```

Comment:
This document is for the Sa language (ISO 639-3: `SAX`).  Sa is an Austronesian language spoken in southern Pentecost Island, Vanuatu.  The links added to this document are not relevant to the Sa language, please remove them.